mrp_production_category_notes/models/mrp_production.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Angel Rivas <angel.rivas@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            product_id = self.env["product.product"].browse(vals.get("product_id"))
+            if product_id and product_id.categ_id:
+                category = product_id.categ_id
+                mrp_notes = category.mrp_notes or category.parent_mrp_notes
+            if mrp_notes:
+                current_notes = vals.get("notes", "").strip()
+                if current_notes and not current_notes == "<p><br></p>":
+                    vals["notes"] += f"\n{mrp_notes}"
```

Comment:
I'm not entirely convinced. Can you run a single if statement to just add the carriage return? You'll always have to specify the value; the only extra thing is the carriage return.